src/main/resources/i18n/strings.properties

```diff
@@ -14,6 +14,7 @@ generic.button.copied=Copied!
 generic.button.done=Done
 generic.button.next=Next
 generic.button.print=Print
+generic.button.reset=Reset
```

Comment:
In general only introduce generic button labels, if they are used more than once or there is other good reasoning for it. (e.g. in the future there is more to reset). Otherwise important context for translation is lost. 

Here it is ...okayish.